### ISSUE TITLE
Need to support other GPIOs for button and LED

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-config#57]
Here we only handle the indoor and outdoor miner buttons...

https://github.com/NebraLtd/hm-config/blob/39c54b2eb0c412f4a84a3248019086c3a7fa2167/config-python/config_program.py#L999-L1008

However in the [variant details file](https://github.com/NebraLtd/helium-hardware-definitions/blob/master/src/hm_hardware_defs/variant.py) we already have the `STATUS` and `BUTTON` values which we can use instead.

**Comment by shawaj** [comment]
Guess we can just change this to:

```
buttonGPIO = variantDetails['BUTTON']
statusGPIO = variantDetails['STATUS']

userButton = Button(buttonGPIO, hold_time=2) 
statusLed = LED(statusGPIO) 
```

**Comment by shawaj** [comment]
Closed by #69
